ods_ci/tests/Resources/Files/pipeline-samples/take_nap.py

```diff
@@ -0,0 +1,32 @@
+from kfp import dsl
+from kfp import components
+
+def take_nap(naptime_secs: int) -> str:
+    """Sleeps for secs"""
+    from time import sleep
+    print(f"Sleeping for {naptime_secs} seconds: Zzzzzz ...")
+    sleep(naptime_secs)
+    return "I'm awake now. Did I snore?"
+    
+def wake_up(message: str):
+    """Wakes up from nap printing a message"""
+    print(message)
+     
+take_nap_op = components.create_component_from_func(
+    take_nap, base_image='registry.redhat.io/ubi8/python-39@sha256:3523b184212e1f2243e76d8094ab52b01ea3015471471290d011625e1763af61')
```

Comment:
Replace with `DataSciencePipelinesKfpTekton.base_image` See an example in https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Files/pipeline-samples/flip_coin.py#L50C31-L50C71

Reply:
Ok, for now. Because this is not being used in automated tests. The way we are compiling is different